packages/breadcrumb/stories/Breadcrumb.stories.tsx

```diff
@@ -0,0 +1,17 @@
+import * as React from "react";
+import { storiesOf } from "@storybook/react";
+import { withReadme } from "storybook-readme";
+import { Breadcrumb } from "../index";
+
+const readme = require("../README.md");
+
+storiesOf("Breadcrumb", module)
+  .addDecorator(withReadme([readme]))
+  .add("default", () => (
+    <Breadcrumb>
+      <span>One</span>
+      <span>
+        T<em>wo</em>
```

Comment:
The T is outside the <em>.

Reply:
This was the intended thin I only wanted to put the `wo` in the `em` this is showing that the `>` is only added to direct children of the Breadcrumb.

Reply:
Ah, ok.